.changes/unreleased/tinymce-TINY-10605-2024-05-03.yaml

```
@@ -0,0 +1,6 @@
+project: tinymce
+kind: Added
+body: Added `options.debug` API
```

```suggestion
body: Added `options.debug` API.
```
Or maybe something like:
```suggestion
body: Added `options.debug` API that logs the initial raw editor options to console.
```
I guess we flush it out in the release notes anyway.